system/system.go

```diff
@@ -143,12 +149,25 @@ func isRpm() bool {
 	}
 
 	// See if it has only one of the package managers
-	if hasCommand("rpm") && !hasCommand("dpkg") {
+	if hasCommand("rpm") && !hasCommand("dpkg") && !hasCommand("apk") {
 		return true
 	}
 	return false
 }
 
+func isAlpine() bool {
+	if _, err := os.Stat("/etc/alpine-release"); err == nil {
+		return true
+	}
+
+	// See if it has only one of the package managers
+	if hasCommand("dpkg") && !hasCommand("rpm") && hasCommand("apk") {
```

Comment:
Does alpine use `dpkg`? If not, the first conditional should be a `!`.

Also can you update the readme limitations section, otherwise looks good.